src/implementation/Client/HTTPClient/HTTPClient.ts

```diff
@@ -16,12 +16,15 @@ export default class HTTPClient implements IClient {
   private readonly httpAgent;
   private readonly httpsAgent;
 
-  constructor(host = "127.0.0.1", port = "50050", options: DaprClientOptions = {
-    isKeepAlive: true
-  }) {
-    this.isInitialized = true;
+  constructor(
+    host = Settings.getDefaultHost()
+    , port = Settings.getDefaultHttpPort()
+    , options: DaprClientOptions = {
+      isKeepAlive: true
+    }
+  ) {
     this.clientHost = host;
-    this.clientPort = port;
+    this.clientPort = port ?? port;
```

Comment:
? What does this do? port or port?

Reply:
Oops, thanks for catching that. Only supposed to be `port`.